corehq/apps/domain/auth.py

```diff
@@ -134,3 +139,8 @@ def _inner(request, *args, **kwargs):
         else:
             return HttpResponse('Inactive user', status=401)
     return _inner
+
+
+def formplayer_auth(view):
+    wrapper = validate_request_hmac('FORMPLAYER_INTERNAL_AUTH_KEY', ignore_if_debug=True)
```

Comment:
is it necessary to default `ignore_if_debug` to True? (could we instead default it to False?)

Reply:
This is intended to make it easy to use formplayer in a dev environment without having to setup the shared key